sdk/services/futures.ts

```diff
@@ -272,6 +272,32 @@ export default class FuturesService {
 		return positions;
 	}
 
+	public async getMarketFundingRates(marketAsset: FuturesMarketAsset) {
+		const periodLength = PERIOD_IN_SECONDS.TWO_WEEKS;
+		const minTimestamp = Math.floor(Date.now() / 1000) - periodLength;
+
+		const response = await request(
+			this.futuresGqlEndpoint,
+			gql`
+				query fundingRateUpdate($marketAsset: Bytes!, $minTimestamp: BigInt!) {
```

Comment:
Could be nice to move this into `queries` as the idea was to try and keep some of this more bloated query string code out of the service files.

Reply:
Moved in the latest commit.